src/dataflow/src/render/sinks.rs

```diff
@@ -158,23 +158,7 @@ where
                 });
                 collection
             }
-            SinkEnvelope::Tail { emit_progress } => keyed
-                .consolidate()
-                .inner
-                .map({
-                    let mut rp = Row::default();
-                    move |((k, v), time, diff)| {
-                        rp.push(Datum::from(numeric::Numeric::from(time)));
-                        if emit_progress {
-                            rp.push(Datum::False);
-                        }
-                        rp.push(Datum::Int64(i64::cast_from(diff)));
-                        rp.extend_by_row(&v);
-                        let v = rp.finish_and_reuse();
-                        ((k, Some(v)), time, 1)
-                    }
-                })
-                .as_collection(),
+            None => keyed.map(|(key, value)| (key, Some(value))),
```

Comment:
I totally get wanting to pull the fake Tail envelope out, but without your git commit comment it's hard to understand what `None` would imply here. It might be helpful to add an inline comment?

Reply:
I'll add a comment. 👍